tap_dbt/streams.py

```diff
@@ -59,7 +59,10 @@ def partitions(self) -> list[dict]:
 
     def get_new_paginator(self) -> DbtPaginator:
         """Return a new paginator instance for this stream."""
-        return DbtPaginator(start_value=0, page_size=100)
+        page_size = 100
+        if page_size := self.config.get("page_size"):
```

Comment:
Unfortunately we still have to support 3.7 versions. Do you mind rewriting without the walrus operator?

Reply:
No worries, I only just learnt this and today I learnt it was a walrus! 🌊